### ISSUE TITLE
sendmail recipient isn't resolved when executed by crontab but works executed manually

### ISSUE THREAD

**Issue by fabienmorin** [thp/urlwatch#284]
When urlwatch is executed via the crontab of my user toto, the mail is not sent and I get an error message in syslog :
`cron[276]: sendmail: 554 5.2.0 STOREDRV.Submission.Exception:InvalidRecipientsException; Failed to process message due to a permanent exception with message Recipient "toto" isn't resolved. All recipients must be resolved before a message can be submitted.`

The strange thing is that if I'm connected with the user toto and launch urlwatch manually, the mail is sent without any problem. Any idea why this is like this and how to get it working with crontab too ?
I had compared the env between the user and the crontab, but it seems not related to that.

I tried the solution of #233 but it doesn't solve the problem. I think it is a different one.
I'm using urlwatch 2.14 on Debian Stretch.

**Comment by fabienmorin** [comment]
up...
no one know how to fix this issue ?

**Comment by thp** [comment]
Can you try something like `toto@localhost` or whatever the system's mailname is?

**Comment by fabienmorin** [comment]
I think I solved the problem. It seem that I had at least two problems in the same time.
- the bad configuration of sendmail and ssmtp. I mainly change /etc/ssmtp/ssmtp.conf (to replace `rewriteDomain=mydomain` into `rewriteDomain=`), but may be I changed other things that impact this problem
- the UnicodeEncodeError: 'ascii' codec problem (#51), but I was not able to simply change the local in the crontab by adding `env LC_ALL=en_US.UTF-8` before the urlwatch command, I did
`sudo dpkg-reconfigure locales`
to chose en_US.UTF-8
Then it works now. But I'm not anymore using my country locale
